bash/solutii/alex-mitan/compare.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIFFERENCES=$(diff $1 $2)
```

Comment:
Se ia path-ul pana la primul spatiu si cel mai probabil imi da "directory not found"?